app/client/ui2018/breadcrumbs.ts

```diff
@@ -82,10 +85,7 @@ interface PartialWorkspace {
   name: string;
 }
 
-const fiddleExplanation = (
-  'You may make edits, but they will create a new copy and will\n' +
-    'not affect the original document.'
-);
+const fiddleExplanation = t('FiddleExplanation');
```

Comment:
We'll need to remove this const and inline the t('FiddleExplanation'). Right now, it's being evaluated before the application has a chance to set things up.